templates/tpos/tpos.html

```diff
@@ -30,168 +23,354 @@ <h5>
-              size="xl"
-              :outline="!($q.dark.isActive)"
-              rounded
-              color="warning"
-              class="btn-cancel"
-              >⬅</q-btn
-            >
-            <q-btn
-              unelevated
-              @click="addAmount()"
-              size="xl"
-              :outline="!($q.dark.isActive)"
-              rounded
-              color="primary"
-              class="btn-plus"
-              >+</q-btn
-            >
-            <q-btn
-              unelevated
-              @click="submitForm()"
-              size="xl"
-              :outline="!($q.dark.isActive)"
-              rounded
-              color="positive"
-              class="btn-confirm"
-              >Ok</q-btn
-            >
+              label="Clear cart"
+              padding="md"
+            ></q-btn>
           </div>
         </div>
+      </q-drawer>
+      <div class="row justify-center q-col-gutter-md q-pa-md">
+        <div class="col-12 col-sm-8 col-md-6">
+          <q-input v-model="searchTerm" label="Filter">
+            <q-icon
+              v-if="searchTerm !== ''"
+              name="close"
+              @click="searchTerm = ''"
+              class="cursor-pointer"
+            />
+          </q-input>
+        </div>
       </div>
-    </q-page-sticky>
-    <q-page-sticky position="top-right" :offset="[18, 18]">
-      <q-btn
-        @click="showLastPayments"
-        fab
-        icon="receipt_long"
-        color="primary"
-      ></q-btn
-    ></q-page-sticky>
-    <q-page-sticky position="top-right" :offset="[18, 90]">
-      <q-btn
-        @click="toggleFullscreen"
-        fab
-        :icon="fullScreenIcon"
-        color="primary"
-      ></q-btn>
-    </q-page-sticky>
-    <q-page-sticky position="top-right" :offset="[18, 162]">
-      <q-btn
-        @click="atm"
-        v-if="withdrawamtposs > 0"
-        fab
-        icon="atm"
-        color="primary"
-      ></q-btn>
-    </q-page-sticky>
+      <div class="row q-col-gutter-md q-pa-md">
+        <div
+          class="col-12 col-sm-6 col-md-4 col-lg-3"
+          v-for="item in filteredItems"
+          :key="item.name"
+        >
+          <q-card class="full-height column">
+            <img
+              v-if="item.image"
+              class="q-pa-md responsive-img"
+              :src="item.image"
+            />
+
+            <q-card-section class="q-mb-auto">
+              <div class="text-h6 ellipsis">${item.title}</div>
+              <div class="text-subtitle1">${item.formattedPrice}</div>
+              <div
+                v-if="item.description"
+                class="text-caption ellipsis-2-lines q-py-sm"
+              >
+                ${item.description}
+              </div>
+            </q-card-section>
+
+            <q-card-actions vertical class="q-pa-md">
+              <q-btn @click="addToCart(item)" padding="md" color="primary"
+                >Add (${item.formattedPrice})</q-btn
```

Comment:
Maybe we do not need to show the price twice?

```suggestion
                >Add</q-btn
```